src/main/java/javax/validation/valueextraction/ValueExtractorDeclarationException.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Bean Validation API
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package javax.validation.valueextraction;
+
+import javax.validation.ValidationException;
+
+/**
+ * Raised if the configuration of {@link ValueExtractor} is illegal, e.g. because multiple extractors for the same type
+ * and type use have been configured in {@code META-INF/validation.xml}.
```

Comment:
Just to be sure, you're just giving an example here and it's intentional you didn't talk about the `ValidatorContext`?